data/drak/indigenous.txt

```diff
@@ -249,6 +249,70 @@ ship "Embershade"
 
 
 
+ship "Embersylph"
+	noun "creature"
+	sprite "ship/embersylph/embersylph"
+		"frame rate" 36
+		"random start frame"
+		"scale" 0.5
+	"never disabled"
+	attributes
+		category "Light Warship"
+		"shields" 7200
+		"hull" 1800
+		"required crew" 1
+		"bunks" 1
+		"mass" 140
+		"drag" 3.2
+		"heat dissipation" .7
+		"cargo space" 0
+		"fuel capacity" 100
+		"thrust" 8
+		"reverse thrust" 3
+		"turn" 90
+		"energy capacity" 800
+		"energy generation" 7
+		"hull repair rate" 2
+		"scram drive" 0.4
+		"jump fuel" 100
+		"hyperdrive" 1
+		"ramscoop" 1.5
+		"gaslining" 1
+		"inscrutable" 1
+		"atmospheric scan" 1000
+		"shield resistance" 1
+	outfits
+		"Absorption Organ" 1
+	turret 0 0 "Absorption Organ"
+	leak "blood" 2
+	explode "blood" 1
+	description "Ember Ray."
+	
+ship "Embersylph" "Embersylph (Small)"
+	sprite "ship/embersylph/embersylph"
+		"scale" 0.5
+	add attributes
+		"shields" -2800
+		"hull" -900
+		"thrust" 2
+		"turn" 10
+
+outfit "Absorption Organ"
+	category "Turrets"
+	thumbnail "outfit/unknown"
+	"turret mounts" -1
+	weapon
+		"velocity" 150
+		"inaccuracy" 20
+		"turret turn" 4
+		"lifetime" 1
+		"reload" 3
+		"energy damage" 80
```

Comment:
```suggestion
		"firing shields" -10
		"energy damage" 80
```

I'm not really sure what an appropriate value is for this, but -10 should be good. So yeah, it literally heals its shields by draining its target. ES doesn't have the means to enforce shields removed = shields gained, so this could theoretically be used to just shoot to heal itself, but the AI isn't that smart, and the player can't get these, so I think it's suitable for a fun odd little detail.

Reply:
The Rulei also has a "gain health" weapon that does the same thing, so this should be fine for now yes.